config.go

```diff
@@ -338,6 +338,8 @@ type Config struct {
 
 	AcceptKeySend bool `long:"accept-keysend" description:"If true, spontaneous payments through keysend will be accepted. [experimental]"`
 
+	AcceptAMP bool `long:"accept-amp" description:"If true, spontaneous payments via AMP will be accepted."`
```

Comment:
should be on by default? :D

Reply:
Even when this is off, we will accept payments to AMP invoices, right?

Reply:
> should be on by default? :D

Much of the feedback we got around keysend seemed to indicate folks would rather it be opt-in, so I just mimicked the behavior with `accept-amp`

> Even when this is off, we will accept payments to AMP invoices, right?

Yes, this only affects the _spontaneous_ case where we JIT insert invoices